service/controller/appcatalog/v1/resource/index/resource.go

```diff
@@ -0,0 +1,80 @@
+package index
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/controller"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	// Name is the identifier of the resource.
+	Name = "indexv1"
+)
+
+// Config represents the configuration used to create a new chart resource.
```

Comment:
```suggestion
// Config represents the configuration used to create a new index resource.
```

Reply:
There are several references to `chart` here. Please make sure they all get updated.